hack/cluster-clean.sh

```diff
@@ -26,7 +26,7 @@ source cluster-up/cluster/$KUBEVIRT_PROVIDER/provider.sh
 source hack/config.sh
 
 function patch_remove_finalizers() {
-    _kubectl patch --type=json -p '[{ "op": "remove", "path": "/metadata/finalizers" }]' $@
+    _kubectl patch --type=json -p '[{ "op": "remove", "path": "/metadata/finalizers" }]' $@ || true
```

Comment:
This looks like it would mask errors. Why is this necessary?

Reply:
We don't need anymore..

`make cluster-clean` was deleting the created VMIs and fixing them to remove them. However, during patching some VMs were actually deleted and then the patch command failed as the obj no longer existed....

Anyway, I will remove this